routes/web/project/github.rb

```diff
@@ -39,5 +39,31 @@ class CloverWeb
         r.redirect "https://github.com/apps/#{Config.github_app_name}/installations/new", 302
       end
     end
+
+    r.on "cache" do
+      r.get true do
+        repository_ids = @project.github_installations_dataset.eager(:repositories).all.flat_map(&:repositories).map(&:id)
```

Comment:
This fetches all the data (e.g. detoasting, busting potential index only scans, etc) in the record from Postgres and makes ruby objects with all of those, and then throws it away for the "id" strings.

```ruby
@project.github_installations_datase.dataset.join(:github_repository, installation_id: :id).select_map(Sequel[:github_repository][:id])
```

But perhaps you can get a similar effect from something like this?  I don't know "eager" as well as "join"

```ruby
@project.github_installations_dataset.eager(:repositories).select_map(Sequel[:github_repository][:id])
```